### PR TITLE
Abass prework

### PR DESCRIPTION
**Pull request by abassawo**
HI guys - I added some pre-work material, including a video on the value of learning to code, some articles on what programmers do, and five basic concepts of programming languages. Would love any comments and feedback on the content.

### CODE REVIEW

**Comment by jsjrobotics** [comment]
@abassawo Please move this into a lesson folder that corresponds to which lesson it is for.

**Comment by abassawo** [comment]
@jsjrobotics @ramonaharrison  I was intending for the prework folder to be assigned prework for the entire unit. For individual lectures, prework readings will be added to their appropriate folders. Let me know if you think this is too much mandatory reading for first day of class.

File: pre-work/assigned_prework.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Assigned Prework
+
+#### Watch the following video
+[Mitch Resnick: Let's teach kids to code](https://www.youtube.com/watch?v=Ok6LbV6bqaE)
```
Comment (ramonaharrison) [inline pre-work/assigned_prework.md]: I enjoyed watching this... Nice pick :+1:

File: pre-work/assigned_prework.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+### What do programmers do?
+
+[What is Programming](http://interactivepython.org/courselib/static/pythonds/Introduction/WhatIsProgramming.html)
+
+[What is Computer Science](http://interactivepython.org/courselib/static/pythonds/Introduction/WhatIsComputerScience.html)
+
+[What does a software engineer do?](http://interactivepython.org/courselib/static/pythonds/Introduction/WhatIsComputerScience.html)
```
Comment (ramonaharrison) [inline pre-work/assigned_prework.md]: This link is the same as the one above

**Comment by ramonaharrison** [comment]
Looks good... I think it's a comfortable amount. Left a couple in-line comments.

**Pull request merged**